database/models/tables.py

```diff
@@ -51,9 +51,26 @@ def _full_search(self, search_keyword):
         abort(404)
 
     def add_new_artist(self, request):
-        
+        #TODO: GenreTable and related ones need to be updated when a new artist is added
+        artist_to_add = request.form.to_dict()
+        statement = (
+            "INSERT INTO artist (name, website, image, location, description) VALUES ('{}', '{}', '{}', '{}', '{}')".format
```

Comment:
Is there a reason you can't do this in query param format?

Reply:
There might be a reason. I ran into a few problems. However, if there is a reason that the query params don't work, can you put in a comment explaining that? i think it might be helpful for both of us to rememebr that

Reply:
I will go in and add a comment myself. The code looks good.